Contest_API.md

```diff
@@ -1178,16 +1178,17 @@ Returned data:
 
 ### Accounts
 
-The accounts used for access to the contest.
+The accounts used for accessing the contest, including the account representing the client.
 
 The following endpoints are associated with accounts:
 
 | Endpoint                       | Mime-type        | Required? | Description
 | :----------------------------- | :--------------- | :-------- | :----------
 | `/contests/<id>/accounts`      | application/json | yes       | JSON array of all accounts with properties as defined in the table below.
 | `/contests/<id>/accounts/<id>` | application/json | yes       | JSON object representing a single account with properties as defined in the table below.
+| `/contests/<id>/account`       | application/json | yes       | JSON object representing a single account with properties as defined in the table below.
```

Comment:
I think it would be good to mention here that this is the account one is logged in with.

Reply:
Fixed in new commit.